pointcloud/basic_octree_utils_test.go

```diff
@@ -247,9 +247,11 @@ func validateMetadata(t *testing.T, bOct *BasicOctree) {
 	test.That(t, bOct.meta.MinY, test.ShouldEqual, metadata.MinY)
 	test.That(t, bOct.meta.MaxZ, test.ShouldEqual, metadata.MaxZ)
 	test.That(t, bOct.meta.MinZ, test.ShouldEqual, metadata.MinZ)
-	test.That(t, bOct.meta.TotalX(), test.ShouldAlmostEqual, metadata.TotalX())
-	test.That(t, bOct.meta.TotalY(), test.ShouldAlmostEqual, metadata.TotalY())
-	test.That(t, bOct.meta.TotalZ(), test.ShouldAlmostEqual, metadata.TotalZ())
+
+	tolerance := 0.0001
```

Comment:
Can we please add a comment why this is needed?

Reply:
done